finder-app/finder.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+# Shell script for Assignment 1
+# Author: Dhiraj Bennadi
+
+
+# $0 represents the first argument to the script, and subsequent numbers represent the next arguments
+echo "Executing Shell script: $0"
+
+if [ $# -lt 2 ]
+then
+    echo "Less number of arguments passed"
```

Comment:
@dhirajbennadi think about how you could make this error message more descriptive for the user...